interfaces/builtin/desktop.go

```diff
@@ -284,12 +287,34 @@ type desktopInterface struct {
 	commonInterface
 }
 
-func (iface *desktopInterface) fontconfigDirs() []string {
+func (iface *desktopInterface) mountFontCache(attribs interfaces.Attrer) (bool, error) {
```

Comment:
Nitpick, perhaps `shouldMountFontCache` so that the name doesn't imply actually mounting anything from snapd.